src/99-appendices/11-arterial-spin-labeling.md

```diff
@@ -0,0 +1,105 @@
+# Appendix XI: Arterial Spin Labeling
+
+These sections provide additional clarification for some specific topics within the BIDS specification for Arterial Spin Labeling.
+
+## Which image is `control` and which is `label`?
+
+The `control` and `label` images are acquired with identical acquisition parameters, except that the blood magnetization flowing into the imaging region is effectively inverted
+in the `label` image compared to the `control` image. In case of doubt, an easy rule of thumb is that the `deltaM`=`control`-`label` subtraction should result in a perfusion-
+weighted image with a positive sign. For example, in the case of FAIR this would be selective inversion (`control`) and non-selective inversion (`label`).
```

Comment:
```suggestion
The `control` and `label` images are acquired with identical acquisition parameters,
except that the blood magnetization flowing into the imaging region is effectively inverted
in the `label` image compared to the `control` image.
In case of doubt, an easy rule of thumb is that the `deltaM`=`control`-`label` subtraction
should result in a perfusion-weighted image with a positive sign.
For example, in the case of FAIR this would be selective inversion (`control`)
and non-selective inversion (`label`).
```